docs/components/Readme.md

```diff
@@ -0,0 +1,33 @@
+[Readme](../../README.md) » BaseComponents
```

Comment:
Sometimes we use _index_, sometimes _README_ - this has to be consistent. The link from docs/index.md goes to docs/components/index.md. I think this file should be renamed to index.md, and also link back to ../index.md.

Reply:
If we'd opt for the README.md way, this file still has to be renamed. Currently it is called Readme.md, so the backlinks do not work on Unix other than MacOS (when run with a case-insensitive file system, which seems to be the default, unfortunately).

Reply:
At the end we will go through all files and adjust all breadcrumbs.